src/cljs/athens/views/blocks/core.cljs

```diff
@@ -204,7 +219,7 @@
                                    (fn [{:block/keys [original-uid uid] :as block}]
                                      (assoc block :block/uid (or original-uid uid)))
                                    block)
-             {:search/keys [] :keys [dragging drag-target]} @state
```

Comment:
Kondo said this `drag-target` was unused so I removed it, but I'm not sure that was the right thing to do.